tools/api-markdown-documenter/examples/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
There are a couple of important aspects to this `examples` config:
2. `paths` - We want our example code to look like _external_ usage would. So, we have aliased the `src` directory with the name of the package such that we can reference the code the same way end-users would.
1. `"noEmit": true` - we don't need to generate any javascript from this code; we strictly want to make sure it builds.

Reply:
An example that wanted to showcase importing from alpha/legacy/another-custom-entrypoint would just use that as the path key and still point to `../src`, right?

Reply:
Confirmed: `..` works correctly for this. I've updated the config.